### ISSUE TITLE
More examples

### ISSUE THREAD

**Issue by nombrekeff** [nombrekeff/cardboard-js#11]

**Comment by Otomatyk** [comment]
Why not add the examples that you showed on dev.to ?

**Comment by nombrekeff** [comment]
Good call. I removed them from the readme because they made it way to messy, I've now added the examples to the [Wiki](https://github.com/nombrekeff/cardboard-js/wiki/Examples). 

There's just 2 examples currently, but I want to add as many as I can. Not only to showcase the project, but also to find bugs, problems, missing features,  etc...

I will update the description and title of this message to better explain what it's about. 

Here's my discord in case you have any question or want help with anything: `nombrekeff`. Feel free to reach out

**Comment by Otomatyk** [comment]
When the bundler is ready (Should I say "When it will be" ?), I use it (For the moment I haven't used it yet, the complexity of the installation was too important for me) and add more example to the wiki or find bug

**Comment by nombrekeff** [comment]
I will take a look at the PR later. I'm now writing a couple of documents for contributing and settings up the project. I will share them here when I finish, they will explain the contributing process and how to setup the project for development!

**Comment by nombrekeff** [comment]
Here, you can find a little guide on [how to contribute](https://github.com/nombrekeff/cardboard-js/wiki/Contributing-Guide), and how to [set up the project for development](https://github.com/nombrekeff/cardboard-js/wiki/Development-Guide).

**Comment by Otomatyk** [comment]
Why you didn’t added the CONTRIBUTING.md file ?

**Comment by nombrekeff** [comment]
I usually do that and might consider adding one. But I don't like how cluttered the project becomes with so many files. It might be a good idea to have it though, as you can check it when working on the project!
